commcare_connect/opportunity/migrations/0025_create_send_inactive_notification_periodic_task.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.5 on 2023-11-02 09:09
+
+from django.db import migrations
+from django_celery_beat.models import IntervalSchedule, PeriodicTask
+
+
+def create_send_inactive_notification_periodic_task(apps, schema_editor):
+    interval, _ = IntervalSchedule.objects.get_or_create(every=1, period=IntervalSchedule.DAYS)
```

Comment:
Just thought of this now. I think it would be better to use a cron schedule so that it fires at a consistent time each day, probably around 7am UTC I think.

Reply:
I added this case. Also refactored some of the logic to make testing easier. https://github.com/dimagi/commcare-connect/pull/181/commits/f629e9ed52c71c65c8e04a078a53ad67e6a89441

Reply:
Done. https://github.com/dimagi/commcare-connect/pull/181/commits/471037d0ba6c721638434fd4748d374f9436f4db